data/content/about.md

```diff
@@ -1,3 +1,51 @@
-Free and guided workshops in Rust – a systems programming language that runs blazingly fast, prevents segfaults, and guarantees thread safety.
+RustBridge is a workshop focused on getting underrepresented people with
+a background in another programming language to learn Rust and join the
```

Comment:
yes, railsbridge has a LOT more content on things like command line, git, text editors, etc that we don't have in our curriculum currently. i'd want to incorporate that before we claim to support nonprogrammers.